javascript/javascript1/week1/housePricerApp.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+
+        //HouseyPrice (A House price estimator)
+
+
+
+function costOfHouse() {
+  let houseWide = prompt("What is the width of your house?");
```

Comment:
Interest use of prompt. I think given the excerize this might be a bit beyond the scope. Did this code execute as expected?

Reply:
yes,.it worked great